content/docs/gke/deploy/deploy-ui.md

```diff
@@ -5,9 +5,12 @@ weight = 3
 +++
 
 This page provides instructions for using the Kubeflow deployment web app to
-deploy Kubeflow on GCP. If you prefer to use the
-command line, see the guide to 
+deploy Kubeflow on GCP. The deployment web app currently supports
+**Kubeflow {{% kf-deployment-ui-version %}}**.
+
+For more control over your deployment, see the guide to 
 [deployment using the CLI](/docs/gke/deploy/deploy-cli).
```

Comment:
```suggestion
[deploy Kubeflow using the CLI](/docs/gke/deploy/deploy-cli).
```

Reply:
Thanks for this suggestion! I don't think the suggested syntax works in this location in the sentence. We need a noun or noun phrase. So, either "deployment" or "deploying". I'll stick with what's there for now.